python/lsst/ts/IntegrationTests/maintel_standby_disabled.py

```diff
@@ -35,11 +35,13 @@ class MainTelStandbyDisabled(BaseScript):
     index: int = 1
     configs: tuple = (
         registry["maintel_standby_disabled"],
+        registry["mtaircomp_standby_disabled"],
```

Comment:
I think we need to take this out since the component goes into DI|SABLED state on start. If we run this test, the command will get rejected causing the script to fail and will pause all other executions.

Reply:
Removed.